packages/lwc/scripts/utils/rollup.js

```diff
@@ -28,11 +28,15 @@ function rollupFeaturesPlugin(prod) {
 function rollupConfig(config) {
     const { input, format, name, prod, target, targetDirectory, dir, debug = false } = config;
     const compatMode = target === 'es5';
+
     return {
         inputOptions: {
             input,
             plugins: [
-                prod && rollupReplace({ 'process.env.NODE_ENV': JSON.stringify('production') }),
+                prod &&
+                    rollupReplace({
+                        'process.env.NODE_ENV': JSON.stringify(debug ? 'prod_debug' : 'production'),
```

Comment:
I'm still confused about why is this here? The description of the PR doesn't tell much.